service/share/autoyast-compat.json

```diff
@@ -362,31 +361,28 @@
       },
       {
         "key": "enable",
-        "support": "yes",
         "children": [
           {
             "key": "service",
-	    "support": "yes"
+	          "support": "yes"
```

Comment:
This (and several more below) looks strange. Some problem with spaces vs tabs?

Reply:
I don't know.  I open the file and it looks good.